src/pages/greeting/greeting.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { HomePage } from '../pages/home/home/../';
+
+/**
+ * Generated class for the GreetingPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-greeting',
+  templateUrl: 'greeting.html',
+})
+export class GreetingPage {
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad GreetingPage');
+  }
+
+  homey() {
+    this.navCtrl.setRoot('HomePage');
```

Comment:
please fix that : 
`this.navCtrl.setRoot('HomePage');`

if HomePage is declare with import, not used singleqoute again. thanks